rxandroidble/src/main/java/com/polidea/rxandroidble/internal/serialization/ClientOperationQueueImpl.java

```diff
@@ -1,9 +1,11 @@
 package com.polidea.rxandroidble.internal.serialization;
 
 import android.support.annotation.RestrictTo;
+
```

Comment:
We should include the formatting configuration into the repository so these kind of changes would not happen.